runtime/aws/Amazon Relational Database Service/check-rds-custom-admin-username.test.ts

```diff
@@ -2,7 +2,7 @@
 import { RDSClient, DescribeDBInstancesCommand, type DBInstance } from "@aws-sdk/client-rds";
 import { mockClient } from "aws-sdk-client-mock";
 import { ComplianceStatus } from "../../types.js";
-import { test, describe, expect, beforeEach } from "bun:test";
```

Comment:
Bun test should work without imports i think and i do not agree with the jest dependency

Reply:
updated